lib/scripts.js

```diff
@@ -82,12 +83,21 @@ exports.paths = {
  */
 exports.run = (script, callback) => {
   childProcess.execFile(script, (error, stdout, stderr) => {
+
+    // Platform scripts might output warnings to `stderr`
+    // that we don't want to take into consideration
+    // if the script exits with an exit code 0.
```

Comment:
IMHO this comment is confusing - any chance you could reword it? Also, perhaps it would make more sense if this comment was moved to just above the `if (stderr.trim().length) {` line?